apps/src/lib/tools/jsinterpreter/JSInterpreter.js

```diff
@@ -245,6 +246,78 @@ export default class JSInterpreter {
     }
   }
 
+  /**
+   * Builds a list of objects that contain all metadata about any functions in
+   * the gived code string. Each object in the returned list has the following
```

Comment:
Can we use the `@return` tag to be consistent with our commenting style and include the expected type for the returned object?